shared/templates/static/ansible/disable_interactive_boot.yml

```diff
@@ -0,0 +1,14 @@
+# platform = Red Hat Enterprise Linux 7,multi_platform_fedora
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name:  Verify that Interactive Boot is Disabled
```

Comment:
should the new title replacement be used?

Reply:
I think we can merge it without it. Next time it would be nice to use it :-)